### PR TITLE
resolves RabbitMQ depenancy failures during setup

### LINKED ISSUE #212: Build Docker images fails with - RabbitMQ install failure - HTTPS Error 403 - Forbidden
For the image: Building and pushing weathervane-rabbitmq image.

```
Step 10/11 : RUN chmod +x /entrypoint.sh && 	chmod +x /usr/local/bin/rabbitmqadmin && 	chmod 600 /var/lib/rabbitmq/.erlang.cookie &&     rpm --import https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc && 	yum install -y erlang &&     yum install -y https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.7.17/rabbitmq-server-3.7.17-1.el7.noarch.rpm && 	rabbitmq-plugins enable rabbitmq_management &&   	yum -y clean all
 ---> Running in d86a987981f1
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: d36uatko69830t.cloudfront.net
 * epel: mirror.freethought-internet.co.uk
 * extras: d36uatko69830t.cloudfront.net
 * updates: d36uatko69830t.cloudfront.net
[91mhttps://dl.bintray.com/rabbitmq-erlang/rpm/erlang/21/el/7/repodata/repomd.xml: [Errno 14] HTTPS Error 403 - Forbidden
Trying other mirror.
To address this issue please refer to the below wiki article

https://wiki.centos.org/yum-errors

If above article doesn't help to resolve this issue please use https://bugs.centos.org/.

[0m[91m

 One of the configured repositories failed (rabbitmq-erlang),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=rabbitmq-erlang ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable rabbitmq-erlang
        or
            subscription-manager repos --disable=rabbitmq-erlang

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:

            yum-config-manager --save --setopt=rabbitmq-erlang.skip_if_unavailable=true

failure: repodata/repomd.xml from rabbitmq-erlang: [Errno 256] No more mirrors to try.
https://dl.bintray.com/rabbitmq-erlang/rpm/erlang/21/el/7/repodata/repomd.xml: [Errno 14] HTTPS Error 403 - Forbidden
[0mThe command '/bin/sh -c chmod +x /entrypoint.sh && 	chmod +x /usr/local/bin/rabbitmqadmin && 	chmod 600 /var/lib/rabbitmq/.erlang.cookie &&     rpm --import https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc && 	yum install -y erlang &&     yum install -y https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.7.17/rabbitmq-server-3.7.17-1.el7.noarch.rpm && 	rabbitmq-plugins enable rabbitmq_management &&   	yum -y clean all' returned a non-zero code: 1

```

### PR DESCRIPTION
**Pull request by saintdle**
Fixes #212

### CODE REVIEW

File: dockerImages/rabbitmq/Dockerfile [hunk 1]
```diff
@@ -2,22 +2,21 @@
...
-COPY isUp.pl /isUp.pl    
+COPY isUp.pl /isUp.pl
 ARG http_proxy
 
 RUN chmod +x /entrypoint.sh && \
 	chmod +x /usr/local/bin/rabbitmqadmin && \
 	chmod 600 /var/lib/rabbitmq/.erlang.cookie && \
     rpm --import https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc && \
-	yum install -y erlang && \
+	yum install -y https://github.com/rabbitmq/erlang-rpm/releases/download/v20.1.7.1/erlang-20.1.7.1-1.el7.centos.x86_64.rpm  && \
```
Comment (hrosenbe) [inline dockerImages/rabbitmq/Dockerfile:17]: Please change this line to:

`yum install -y https://github.com/rabbitmq/erlang-rpm/releases/download/v21.3.8.21/erlang-21.3.8.21-1.el7.x86_64.rpm && \`

This gives the same version of erlang as was previously used.  The version that you are using here is too old for the version of RabbitMQ.

File: dockerImages/rabbitmq/Dockerfile [hunk 2]
```diff
@@ -2,22 +2,21 @@
 # SPDX-License-Identifier: BSD-2-Clause
 FROM namespace/weathervane-centos7:version
 MAINTAINER hrosenbe@vmware.com
-	
-COPY erlang.repo /etc/yum.repos.d/erlang.repo
+
```
Comment (hrosenbe) [inline dockerImages/rabbitmq/Dockerfile:5]: Please remove this extra line

**Review by hrosenbe — CHANGES_REQUESTED** [review]
We can accept this pull request once the requested changes have been made.  Thanks for submitting this change.

**Review by saintdle — COMMENTED** [review]
Changes made

**Review by hrosenbe — APPROVED** [review]

**Pull request merged**